<?xml version="1.0"?>
<launch>
    <node name="config_parser_client" pkg="config_parser" type="config_parser_client" output="screen"/>
</launch>